src/tests/test_experiments.py

```diff
@@ -244,6 +263,7 @@ def test_restart_experiment(experiment_class):
                 db_engine=db_engine,
                 project_path=os.path.join(temp_dir, "inspections"),
                 cleanup=True,
+                skip_validation=True,
```

Comment:
Did you add the skip_validation because these are failing? That would be a problem that we should fix on its own. I'd rather most of the tests have as vanilla an Experiment as possible, so if a vanilla Experiment now includes validation, then they should validate. I don't think these little test experiment validations should really take long enough for it to be a major performance issue

Reply:
yeah, several of them were failing and I assumed the tests were probably passing simplified config files just to get at the pieces the were testing. Should we try to fix these here or in a separate PR?

Reply:
in particular, I'm not sure I understand the purposes of the `test_partial_experiments.py` tests and why they're explicitly subsetting down `sample_config()`

Reply:
The partial experiments are because of a request to make partial experiments do the partial work as opposed to just croaking. But it makes sense that they should not pass validation, so the skip_validation flag makes sense.

Could you modify the 'Running parts of an experiment' section of docs/sources/experiments/running.md to say that you should skip validation when calling run on a partial experiment?

And more generally, can you modify the 'Validation an Experiment' section of that same document to reflect the new behavior?